### PR TITLE
Update vector.md

### PR DESCRIPTION
**Pull request by markooff**
**When merged this pull request will:**
- add the section called **3. Resources** to docs/wiki/feature/vector.md 
- modify the name of the section previously called **3. Dependencies** to **4. Dependencies**
- Fill the section **3. Resources** with the information about the polish translation of the document together with the direct link to it

### CODE REVIEW

**Comment by bux** [comment]
PR description is lacking

File: docs/wiki/feature/vector.md [hunk 1]
```diff
@@ -91,6 +91,13 @@ The Vector is controlled with 2 keys: the azimuth key and the range key; <kbd>Ta
 - Press <kbd>R</kbd> until the desired units are displayed.
 - Tap <kbd>Tab&nbsp;↹</kbd> five times fast to save the selection.
 
-## 3. Dependencies
+## 3. Resources
+
+Polish version of this manual with some comments and small additional section
+/polska wersja tego dokumentu opatrzona komentarzami oraz niewielkim dodatkiem:
+
+http://http://pgm.armaonline.pl/forum/viewtopic.php?f=15&t=548
```
Comment (dedmen) [inline docs/wiki/feature/vector.md]: Link is invalid.

**Comment by dedmen** [comment]
We do everything in english. If we start linking to external polish versions we gotta do the same for all the other languages and all the other docs.

**Comment by markooff** [comment]
But while we have some materials (like manualls , FAQs, or even books) already written in one language - why do not share it with the publicity ? If you want to make another material , you can just do it :)

**Comment by markooff** [comment]
@bux - could You precise what does "PR" mean ?

**Comment by jonpas** [comment]
@dedmen this was approved, anyone with their own resource can add links to our docs.

@markooff he means the description of the pull request. "When merged this pull request will:....", you didn't fill it out.

**Comment by markooff** [comment]
OK, I've just filled out the description of the pull request . 
I'm sorry for my missunderstanding this earlier !

ThX, @jonpas :)

**Comment by bux** [comment]
@dedmen ’s point is definitely valid. Though on the other hand we even have translated versions of our repo’s readme file in git. Apart from the acemod page. I wouldn’t know where else to put ace related resources (even in other languages).
But then again... would we link each and every ace tutorial in different languages on YouTube? I doubt so.

Here we link to a Spanish guide:
https://ace3mod.com/wiki/development/how-to-translate-ace3.html

**Comment by dedmen** [comment]
I would prefer if we could directly integrate translations into the documentation. Just add a language selector.
But how do we make sure translations are kept up to date?

**Comment by jonpas** [comment]
You don't.

**Comment by markooff** [comment]
@dedmen I understand your point of view but - there's another which says: "The whole documentation is only in english ... Well, too bad" This point of view is spread between lots people who don't know english as well as you do, but also want to play Arma in their own languages (as you may find it out - ArmA is translated and localised very well ).... :) 
There are a lots of people in many countries ... So that I think that if we (I say "we" as a humble contributor in such great project like ACE3 ) so if we want to go further in popularisation of ACE3 mod - we can turn into translated version - with the ACE3 Official Manual - or at least let the people do this and link the parts of it, translated on their own ... 
This can be the first step, I think. 

And the second problem 
@bux - we dont have to check the actuality of linked materials at all . 
That's because of the internal structure of WIKI ACE3 ! 

The only place to put the links to EXTERNAL CONTENT - is the x. RESOURCES section .
which can be even changed into the "x. ADDITIONAL RESOURCES " section 
This can reveal all the problems - because if they are treat as EXTERNAL RESOURCES - the responsibility for keeping them up to date obviously lays on their authors ! 
And no one else :)

Tell me - how many times You (or someone else ) checked recently the "being up to date " and being accurate to the ACE3 capabilities -  of the 
materials which are linked in section 4 of the ATragMX-ACE3 documentation ?

"4. Official Manual and Horus Videos
    Official Manual
    Horus video part1
    Horus video part2"

I may reply (instead of You) - 0 times. 
And why ? 

Because of the same reason - the fact that those materials are only LINKED EXTERNALLY  and the responsibility is laid only on the Author(s) ....

So - in my opinion - we could do the same with (we could treat in the same way) EXTERNALLY LINKED - translations of the parts of the ACE3 Wiki. Of course only if we put the clear and obvious info " Hey people - those materials ARE EXTERNAL and here there are their authors"...

**Comment by dedmen** [comment]
>Tell me - how many times You (or someone else ) checked recently the "being up to date " and being accurate with the ACE3 capabilities - of the
materials which are linked in section 4 of the ATragMX-ACE3 documentation ?

The ACE AtragMX is built after these things linked there. We don't need to check them.
The pdf is in the ACE repo so we will know when that changes before it does.
And the videos are on youtube and can't be changed out for any other video.

>Because of the same reason - the fact that those materials are only LINKED EXTERNALLY

The youtube videos are external yes. But they cannot change without changing the video ID and invalidating the Link.

Bad example ^^. And I understand that external materials aren't checked by us. I never said that we have to check external stuff. I was only talking about internal.

Just would prefer to have it internally or atleast in a Git repo somewhere. That way people can atleast contribute and update it if anything changes. 
Also if people, that right now already do translations for us, see that we also accept/need translations for the official documentation, they might also want to help with that and we'd get much more translation done than just some links to some external sources.

**Comment by markooff** [comment]
Yes, maybe You're right...

But as usuwaly - there are two different way of deal with this problem : one - to keep everything close, pure and brand-made (as the biggest company/corporation used to do ) , and the other - to let the people make some their own work and efford to cooperate, and the only thing is to make the good (suitable) place for them to to that ....
Maybe there would be much more participants - if they can easily link/send their own materialls , translations etc without need of registering on github without the knowledge of git technology, because they are - in the majority - normal people , not coders (as we mostly are) ...

It is not my duty (fortunatelly) :) - to choose which way is the better (or even the best). 

But , in fact, for the last years , I've never seen those hundreads of thousands of people , staing the whole night at the door and begging "let us translate something, let us translate .... "

**Comment by bux** [comment]
I think it’s “Save enough to try”.

**Comment by bux** [comment]
I’ll approve it if you fix the invalid link

**Comment by Laid3acK** [comment]
The videos in the AtragMx Wiki came from the official Horus YouTube channel. 
Sadly, Horus has cleaned his channel a few months ago.
 I didn't update this wiki yet because i expect find another links ... without success so far.
But it was official resources and i think it's an important point.

**Pull request merged**